test/e2e/gameserver_test.go

```diff
@@ -301,7 +301,7 @@ func TestGameServerRestartBeforeReadyCrash(t *testing.T) {
 		return false
 	})
 	if err != nil {
-		assert.FailNowf(t, "Could not make GameServer Ready: %v", err.Error())
+		assert.FailNowf(t, "Could not make GameServer Ready", "reason: %v", err.Error())
```

Comment:
should there be extra quotes around the comma?

Reply:
@aLekSer pointed this out to me on a previous PR, because the error string wasn't coming through very well. The signature for FailNowf is:

`func FailNowf(t TestingT, failureMessage string, msg string, args ...interface{}) bool`

So the `failureMessage` won't parse any `Sprintf` style injection, only the `msg` argument will.

Reply:
Weird, that's not what I expected based on X...f functions in other packages.